### DISCUSSION TITLE
Improve Project Manager initial empty state

### DISCUSSION THREAD

**Discussion by JohnVeness** [godotengine/godot-proposals#6427] [Editor]
I'd like to suggest an improvement that would improve the experience for first time users.

## Current behavior

At the moment, when you open the Project Manager while having no projects listed (i.e. when you very first start using Godot), a dialog saying "You currently don't have any projects. Would you like to explore official example projects in the Asset Library?". Personally I find this dialog rather messy/cluttery/unfriendly.

![godot-noprojects](https://user-images.githubusercontent.com/2512915/222921850-5fd93aad-8374-48d8-a5a8-042072dd65d5.png)

## Proposed behavior
Instead of the dialog box, I suggest helpful/welcoming text should be placed within the Project Manager window itself, where the list of projects would normally appear, when there are currently no projects.

The text could read maybe something like "Welcome to Godot. Here you can start a New Project, Import an existing project, or view the Asset Library for example projects". This text would be text-only, not clickable, to encourage the user to use the existing buttons/tabs in the window.

This UI pattern of displaying text where there would otherwise be an empty list is used in lots of apps, see for example:
![dolphin-no-files](https://user-images.githubusercontent.com/2512915/222922086-325725d8-817e-4962-a15e-4e42448db78d.png)
![dolphin-files](https://user-images.githubusercontent.com/2512915/222922057-9cbecb41-40c8-4951-a105-32e04e09aef8.png)
![lo-no-recents](https://user-images.githubusercontent.com/2512915/222922098-1a7f87f8-1f7d-447c-aa90-f1221b5af681.png)
![lo-recents](https://user-images.githubusercontent.com/2512915/222922092-c93efaee-260c-49d7-83b0-47a59ea27551.png)

Thank you for reading.

**Comment by Calinou** [comment]
This should be fairly easy to do by displaying a CenterContainer with a Label as a substitute for the ItemList if there are no projects in the list (and hiding this CenterContainer once a project appears in the list).

Feel free to open a pull request for this :slightly_smiling_face:
